index.html

```diff
@@ -0,0 +1,229 @@
+            </div>
+
+            <div class="container1">
+                <div id="count">7,146 views <span id="dot">.</span>  Aug 12, 2020
+            </div>
+
+            <div class="list-group">
+                    <a class="list-group-item" href="#"><i class="fa fa-thumbs-o-up" aria-hidden="true" fa-lg></i>&nbsp;
+                        100</a> &nbsp;
+                    <a class="list-group-item" href="#"><i class="fa fa-thumbs-o-down" aria-hidden="true"
+                            fa-lg></i>&nbsp; 0</a>&nbsp;
+                    <a class="list-group-item" href="#"><i class="fa fa-share" aria-hidden="true"></i>&nbsp;
+                        SHARE</a>&nbsp;
+                    <a class="list-group-item" href="#"><i class="fa fa-floppy-o" aria-hidden="true"></i>&nbsp;
+                        SAVE</a>&nbsp;
+                    <a class="list-group-item" href="#"><i class="fa fa-ellipsis-h" aria-hidden="true"></i></a>
+            </div>
+        </div>
+        <hr id="line1">
+
+        <div class='container2'>
+            
+                <div class="img">
+                    <img src="images/image.png" alt="Youtube_Profile" class="details">
+                </div>
+
+                <div class="fb">
+                    <h4>Famous Youtuber</h4>
+                    <div id="subscribers"> 1Mn Subscribers</div>
+                </div>
+
+                <div class="button1">
+                    <button id="subsribe">SUBSCRIBE</button>
+                </div>
+
+                <div class="container3">
+                    <p id="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius architecto quod, dolore sapiente veritatis necessitatibus quae corrupti quos hic consectetur voluptates eveniet molestiae tempore praesentium, minima cumque accusantium quia. Pariatur. <br> <br>
+                    <a href="#"id="showMore">SHOW MORE</a>
+                    </p>
+                </div>
+            
+        </div>
+        <hr id="line1">
+
+            <div class="sortComments">
+                <p id="usersComments"> 117 Comments</p>
+                <a class="list-group-item" href="#"><i class="fa fa-sort-amount-desc" aria-hidden="true"
+                        fa-lg></i>&nbsp; <span id="SortBy">SORT BY</span> </a>
+            </div>
+
+
+            <div class="container4">
+                <div class="addcomments">
+                    <img id="userImage" src="images/image.png" alt="user comment">
+                    <h4 id="publicComment">Add a public comment....
+                        <hr id="line2">
+                    </h4>
+                </div>
+                   
+                <div class="button">
+                    <button type="button" id="cancel">CANCEL</button>
+                    <button type="button" id="comment">COMMENT</button>
+                </div>
+            </div>
+        </div>
+   
+
+                            <!--SIDEBAR-->
+<div class="sidebar">
```

Comment:
Use semantic HTML: here you should look for something more semantic than a `<div>`, you can check [this](https://www.w3schools.com/html/html5_semantic_elements.asp) link.